components/ContractViewer/ContractCodeEditor.tsx

```diff
@@ -5,7 +5,7 @@ import { useTheme } from 'next-themes'
 
 const ContractCodeEditor = ({ code, line, column, ...props }: any) => {
   const [codeEditor, setCodeEditor] = useState<any>(null)
-  const { theme } = useTheme()
+  const { resolvedTheme } = useTheme()
```

Comment:
I read into it and its weird that `theme` doesn't react to system preference, but whatevs. Thanks for the fix

Reply:
Yeah, I also found that a weird api design  🤷‍♂️